client-appimage/Dockerfile

```diff
@@ -27,7 +29,6 @@ RUN sed -i '/deb-src/s/^# //' /etc/apt/sources.list && \
     # Install packages
     apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install -q -y \
-        libsqlite3-dev \
```

Comment:
I would probably do it in another way
`apt-get build-dep -y libqt5core5a` may install it again
so I would remove it at the same time we remove qt5 base dev package
see `apt-get remove -q -y --purge qtbase5-dev`